azure_jumpstart_arcbox/artifacts/DataOpsLogonScript.ps1

```diff
@@ -170,6 +170,7 @@ foreach ($cluster in $clusters) {
     }
 }
 
+Stop-Transcript
```

Comment:
why are we stopping the transcript here, and then stopping it again on line 261?

Reply:
then on line 182 we are writing to host, but it wont be captured yet because nothing is being transcribed

Reply:
if this is by design thats fine, just looks odd

Reply:
just removed an unnecessary stop-transcript. There is the main transcript which stops before the loop with the jobs, and in each job there is a transcript that starts/stops to log information for each cluster/instance in their own log file.